Fixed #35606 -- Admin changelist broken for model instances with __html__ method. (#18374)
django/django: docs/releases/5.0.8.txt

Bugfixes
========

> * Fixed a regression in Django 5.0 where ``ModelAdmin.action_checkbox`` had the
>   possibility to break admin changelist by returning unsafe characters from
>   model instance's ``__html__`` method. (:ticket:`35606`).

Reviewer: ```suggestion
* Fixed a regression in Django 5.0 where ``ModelAdmin.action_checkbox`` could
  break the admin changelist HTML page when rendering a model instance with a
  ``__html__`` method (:ticket:`35606`).
```
(maybe 🤔)
